api.go

```diff
@@ -278,21 +279,33 @@ type (
 		CatalystPipelineStrategy string `json:"catalystPipelineStrategy,omitempty"`
 	}
 
+	CustomParams struct {
+		URL     string            `json:"url"`
+		Method  string            `json:"method,omitempty"`
+		Headers map[string]string `json:"headers,omitempty"`
+	}
+
+	IPFSParams struct {
```

Comment:
**nit:** WDYT of prepending the names of these structs with `Export` or `ExportTask` so they're clearer? Especially the custom one (could be `CustomExportParams` as well)

Reply:
Updated in 6a485c292676b7cdf82dcb5d1cf6d215c4e50359